src/challenges/challengeOne/5.Modal/Modal.tsx

```diff
@@ -22,7 +22,11 @@ export function Modal({
       <div className="modal w-[300px] rounded-md bg-white p-6">
         <div className="flex items-center justify-between pb-6">
           <h2 className="text-lg font-semibold">{headerText}</h2>
-          <button onClick={onClose} type="button">
+          <button
+            onClick={onClose}
+            type="button"
+            data-testid="modal-close-icon"
```

Comment:
You could have used `aria-label="close"` and added accessibility instead of a test id to query your button